auth/providers/azure/graph/graph.go

```diff
@@ -72,7 +72,7 @@ var (
 )
 
 const (
-	expiryDelta            = 60 * time.Second
+	expiryDelta            = 300 * time.Second
```

Comment:
rename to `tokenExpiryDelta` to be specific?

Reply:
This is not a new var, but I can add a comment?

Reply:
Updated to suggested name